Makefile

```diff
@@ -210,6 +210,10 @@ verify-deps:
 	go mod tidy
 	@git diff --exit-code -- go.mod go.sum
 
+.PHONY: verify-generated
+verify-generated: $(SRC_DEPS)
```

Comment:
There is no benefit of checking all files as it hides original intention. 
Let's just focus on verifying the generated files `pkg/api/generated/openapi/zz_generated.openapi.go`, `pkg/scraper/types_easyjson.go`.